bot/bot.py

```diff
@@ -0,0 +1,23 @@
+import logging
+
+from disnake.ext import commands
+
+from bot import constants
+
+log = logging.getLogger(__name__)
+
+
+class SirRobin(commands.Bot):
+    """Sir-Robin core."""
+
+    def add_cog(self, cog: commands.Cog) -> None:
+        """
+        Delegate to super to register `cog`.
+
+        This only serves to make the info log, so that extensions don't have to.
+        """
+        super().add_cog(cog)
+        log.info(f"Cog loaded: {cog.qualified_name}")
+
+
+bot = SirRobin(command_prefix=constants.SIR_ROBIN_PREFIX)
```

Comment:
See the comment below.
```suggestion
bot = SirRobin(command_prefix=constants.Client.prefix)
```

Reply:
[a8fc686](https://github.com/python-discord/sir-robin/pull/10/commits/a8fc686e88045f580c583045557690e7611d7e08)